programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,4 @@
+function solution(seoul) {
+    return "김서방은 " + seoul.indexOf("Kim") + "에 있다";
+}
```

Comment:
""  보다는 1옆에있는 백택이라 불리는  `` 을 사용하는게 보기좋고 본인이 헷갈리지도 않는다
```suggestion
   return `김서방은 ${seoul.indexOf("Kim")}에 있다`;
}
```